### PR TITLE
Added route for '/help/typeofhelp'

### LINKED ISSUE #537: Implement Help categories on Frontend
Based on design in https://github.com/mozilla/network-pulse/issues/521 relies on form updates in #533 and post detail page layout https://github.com/mozilla/network-pulse/issues/520

Content includes:
- "Get Involved" title
- all "Get Involved" body copy
- a link URL
- tags

https://redpen.io/p/of380006b3156e2c51
desktop details latest 2x

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #537

### CODE REVIEW

File: pages/tag-typy-page.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from "react-helmet";
+import ProjectLoader from '../components/project-loader/project-loader.jsx';
+
+// For pages that display a list of entries based on single param search in the API call
+// e.g., 'tag', 'help_type', ...etc
+class TagTypyPage extends React.Component {
```
Comment (Pomax) [inline pages/tag-typy-page.jsx]: What does Typy mean? Can we name this after "what it is" (even if that's a slightly longer name?)

**Comment by mmmavis** [comment]
@alanmoo you can just take a look at the 2nd commit (for renaming things)

**Comment by mmmavis** [comment]
@alanmoo 

Confirmed that the bug came from API side. Ticket has been filed https://github.com/mozilla/network-pulse-api/issues/174 .

**Comment by mmmavis** [comment]
@Pomax @alanmoo PR has been updated! It turned out I did need to update this PR to make sure query params were encoded.

File: js/service.js [hunk 2]
```diff
@@ -52,6 +52,8 @@ function getDataFromURL(route, params = {}, token = {}) {
 
   Object.assign(params, defaultParams);
 
+  console.log(`params`, params);
```
Comment (alanmoo) [inline js/service.js]: Remove this

**Pull request merged**